wallet/test/run-db.c

```diff
@@ -122,6 +123,7 @@ int main(void)
 	bool ok = true;
 	/* Dummy for migration hooks */
 	struct lightningd *ld = tal(NULL, struct lightningd);
+	ld->config = testnet_config;
```

Comment:
```suggestion
	ld->config = {
	.locktime_blocks = 6 * 24,
	.locktime_max = 14 * 24 * 6,
	...
	};
```

Reply:
example of inline config definition

Reply:
@niftynei Yes I know. I tried to reduce code duplication an need for changes at three locations (main, test and now also stub) if config values are added.

To the context: the new database migration routines allow a `*ld` pointer to have context sensitive migrations. They are needed i.e. for the setchannelfee PR. Currently they need to access at least a config section (maybe more for future migrations). They caused a valgrind complaint as the wallet db tests tried to migrate with an uninitialized config section.

I thought it would be good to have reusable test stubs, thats why I did this...

Reply:
Will push an update tomorrow...

Reply:
cool, sg. right, instead of inlining them, the more sensible/DRY'er approach would be to have a single 'canonical' test config that's defined and reused within the context of the tests, but not totally sure that we've got that type of dependency infrastructure defined for tests at the moment....

Reply:
addressed a8d5189f76384c00797cfb67934825d1dadec8bd